src/command-handler.ts

```diff
@@ -31,6 +41,31 @@ class CommandHandler {
         }
     }
 
+    static isAdmin(interaction: Interaction) {
+        return interaction.memberPermissions?.has(PermissionFlagsBits.Administrator)
+            || state.isAdmin(interaction.user.id);
+    }
+
+    static assertIsPrivilegedRole(interaction: Interaction) {
```

Comment:
Naming doesn't make sense. Shouldn't is be `assertHasPrivilegedRole`? Or if we go with the approach of not requiring to set roles, then just `assertIsPrivileged`?